romancal/datamodels/ramp.py

```diff
@@ -0,0 +1,39 @@
+from .core import RomanDataModel
+
+class RampModel(RomanDataModel):
+    """
+    A data model for 4D ramps.
+
+    Parameters
+    __________
+    data : numpy float32 array
+         The science data
+
+    pixeldq : numpy uint32 array
+         2-D data quality array for all planes
+
+    groupdq : numpy uint8 array
+         3-D data quality array for each plane
+
+    err : numpy float32 array
+         Error array
+
+    zeroframe : numpy float32 array
+         Zeroframe array
+
+    group : numpy table
+         group parameters table
+
+    int_times : numpy table
+         table of times for each integration
```

Comment:
With one integration does this need to be a table?

Reply:
Good catch, this was actually removed - as it was for others. Deleted.